Text/CustomEntitySearch/WebAPISkillContract.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace AzureCognitiveSearch.PowerSkills.Common
+{
```

Comment:
don't copy these files here, just link to them

Reply:
same for WebAPISkillHelper.cs

---
In reply to: [304669534](https://github.com/Azure-Samples/azure-search-power-skills/pull/1#discussion_r304669534) [](ancestors = 304669534)